File: server/entity/Nazionalita.php
<?php

class Nazionalita {

    private $abbreviazione;
    private $valore;

    public function __construct() {

    }

    public function getAbbreviazione() {
        return $this->abbreviazione;
    }

    public function setAbbreviazione($abbreviazione) {
        $this->abbreviazione = $abbreviazione;
    }

    public function getValore() {
        return $this->valore;
    }

    public function setValore($valore) {
        $this->valore = $valore;
    }

    function toArray() {
        return array(
            "abbreviazione" => $this->abbreviazione,
            "valore" => $this->valore
        );
    }
}

File: java/src/entities/Nazionalita.java

package entities;

/**
 * Classe che rappresenta la nazionalita' di un'Anagrafica
 * @author emanuelegargiulo
 */
public class Nazionalita {
    
    private String abbreviazione;
    private String valore;

    /**
     *
     */
    public Nazionalita() {
    }

    /**
     *
     * @return
     */
    public String getAbbreviazione() {
        return abbreviazione;
    }

    /**
     *
     * @param abbreviazione
     */
    public void setAbbreviazione(String abbreviazione) {
        this.abbreviazione = abbreviazione;
    }

    /**
     *
     * @return
     */
    public String getValore() {
        return valore;
    }

    /**
     *
     * @param valore
     */
    public void setValore(String valore) {
        this.valore = valore;
    }
    
}
